CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # Changelog
 
+## 13.3.7 (08/29/23)
+
+* Added bblp custom theme and logos [#31149](https://github.com/gravitational/teleport/pull/31149)
+* Set `127.0.0.1:3080` as Vite default proxy target [#31148](https://github.com/gravitational/teleport/pull/31148)
+* `tsh request drop` behavior was restored to pre-v13.3.5 state where after dropping an access request, the role list will be based on the current role list of the user on the backend, not on the login state (as introduced in #30628) [#31136](https://github.com/gravitational/teleport/pull/31136)
```

Comment:
I would drop this too. It was not an intentional behavior change, and we're just restoring the behavior that only existed in one (buggy) release.